content/blog/chif-driver-not-found.md

```diff
@@ -0,0 +1,117 @@
+
+![Successful in-band GET command](/img/successfulinbandget.png "Successful in-band GET command")
+
+## When do you get the Chif Driver not found error ?
+
+The most typical situation is when you are logged in, as a privileged user in a non-iLO based server, a virtual machine or a laptop, and you issue an iLOrest command or launch a Python or PowerShell script performing in-band Redfish commands.
+
+In those cases, there is no iLO underneath the operating system. Hence the CHIF driver, although present as a file somewhere on disk, cannot not be properly loaded by the application.
+
+### Issuing iLOrest in-band commands on a non-iLO based server
+
+The most typical iLOrest invocation is:
+
+```shell
+ilorest login <RedfishService-IP-address>
+ilorest cmd1
+ilorest cmd2
+...
+ilorest logout
+```
+
+If you don't provide any `<RedfishService-IP-address>` to the `iLOrest login` command, or if you omit completely this command, iLOrest uses the `blobstore://.` target URL and tries to connect to the local iLO via the CHIF driver.
+
+If you use a [Redfish client](https://youtu.be/ur9UKRV_0S8) different from iLOrest that uses another library like the [DMTF Python Redfish Library](https://github.com/DMTF/python-redfish-library), you will not be able to perform in-band management through the CHIF driver. An alternative is to enable the [iLO Virtual NIC](https://www.youtube.com/watch?v=KM1FZ-AlctA) and use its IP address as target URL.
+
+The following picture shows an SSH privileged session into a VMware virtual machine and an iLOrest in-band GET command returning the error. 
+
+![Unsuccessful in-band GET from Virtual Machine](/img/unsuccessfulinbandgetinvm.png "Unsuccessful in-band GET from Virtual Machine")
+
+The next screenshot shows the same iLOrest command launched from a Microsoft Windows laptop.
+
+![Unsuccessful in-band GET from laptop](/img/unsuccessfulinbandgetinwinlaptop.png "Unsuccessful in-band GET from laptop")
+
+The following paragraphs address In-band Python scripts, PowerShell Cmdlets and VMware infrastructures with respect to the CHIF driver. 
+
+### In-band Python scripts
+
+The `blobstore://.` URL mentioned above, can be used as well, in Redfish Python scripts based upon the [HPE python-ilorest-library](https://github.com/HewlettPackard/python-ilorest-library) to perform in-band management operations. Such programs will have the same behavior as iLOrest since iLOrest uses as well the HPE Python Redfish library. 
+
+In the following picture you can see the execution in a virtual machine, of the [get_ilo_ip.py](https://github.com/HewlettPackard/python-ilorest-library/blob/master/examples/Redfish/get_ilo_ip.py) python example configured for in-band management.
```

Comment:
The hotlink with underscore "_" is not rendered correctly in the CMS. I believe "_" is a special character and should be treated as such. You may want to try an escape character (\) before the special character _.

try this and see if it makes the trick: 
[get\_ilo\_ip.py](https://github.com/HewlettPackard/python-ilorest-library/blob/master/examples/Redfish/get_ilo_ip.py)